Composer/packages/server/src/services/project.ts

```diff
@@ -562,7 +563,7 @@ export class BotProjectService {
         const id = await BotProjectService.openProject(
           { storageId: rootBot?.storageId, path: rootBot.path },
           user,
-          true
+          creatingRootBot ?? true
```

Comment:
You should just give `creatingRootBot` a default value, so you don't have to worry about it being `undefined`. This is difficult to understand. 

```suggestion
          creatingRootBot
```

Reply:
Good idea. I'll add that in.